### PR TITLE
Adviser trigger message v4

### PR DESCRIPTION
**Pull request by fridex**
## Related Issues and Dependencies

Related: https://github.com/thoth-station/messaging/pull/390/

## This introduces a breaking change

- [x] No

### CODE REVIEW

File: thoth/investigator/adviser_trigger/investigate_adviser_trigger.py [hunk 1]
```diff
@@ -34,7 +34,7 @@
 _LOGGER = logging.getLogger(__name__)
 
 
-@register_handler(AdviserTriggerMessage().topic_name, ["v1", "v2", "v3"])
+@register_handler(AdviserTriggerMessage().topic_name, ["v1", "v2", "v3", "4"])
```
Comment (harshad16) [inline thoth/investigator/adviser_trigger/investigate_adviser_trigger.py:37]:
```suggestion
@register_handler(AdviserTriggerMessage().topic_name, ["v1", "v2", "v3", "v4"])
```

**Review by harshad16 — CHANGES_REQUESTED** [review]
minor typo

File: thoth/investigator/adviser_trigger/investigate_adviser_trigger.py (hunk 1 above)
Comment (fridex) [inline thoth/investigator/adviser_trigger/investigate_adviser_trigger.py:37]: I wonder if we should keep the older versions there? I think the implementation raises KeyError for older messages as "authenticated" will not be present there.

Comment (fridex) [reply thoth/investigator/adviser_trigger/investigate_adviser_trigger.py]: nice, thanks 👍🏻

Comment (harshad16) [reply thoth/investigator/adviser_trigger/investigate_adviser_trigger.py]: yes let use , `.get()` to keep it back compatible

**Review by harshad16 — COMMENTED** [review]
/lgtm 
:+1:

**Comment by KPostOffice** [comment]
Can you also add provenance in this PR

**Review by KPostOffice — APPROVED** [review]

**Pull request merged**